cypress/e2e/33_orgBountyStatusFilter.cy.ts

```diff
@@ -0,0 +1,255 @@
+
+    const bounty8: Cypress.Bounty = {
+      organization: OrgName,
+      title: 'Bounty8(paid)',
+      category: 'Web development',
+      coding_language: ['Typescript', 'Javascript', 'Lightning'],
+      description: 'This is available',
+      amount: '123',
+      tribe: 'Amazing Org Tribe',
+      estimate_session_length: 'Less than 3 hour',
+      estimate_completion_date: '09/09/2024',
+      deliverables: 'We are good to go man',
+      assign: 'alice'
+    };
+
+    cy.create_bounty(bounty1);
+    cy.wait(1000);
+
+    cy.contains(bounty1.title);
+    cy.wait(5000);
+
+    cy.contains(org.name);
+    cy.wait(1000);
+
+    cy.contains(org.name).invoke('removeAttr', 'target').click({ force: true });
+    cy.wait(5000);
+
+    cy.url().should('include', '/org/bounties');
+    cy.wait(5000);
+
+    cy.create_orgBounty(bounty2);
+    cy.wait(1000);
+
+    cy.create_orgBounty(bounty3);
+    cy.wait(1000);
+
+    cy.contains('Status').first().click();
+    cy.wait(1000);
+
+    cy.contains('label', 'Open').click();
+    cy.wait(1000);
+
+    cy.contains(bounty1.title);
+    cy.wait(1000);
+
+    cy.contains(bounty2.title);
+    cy.wait(1000);
+
+    cy.contains(bounty3.title);
+    cy.wait(1000);
+
+    cy.contains('label', 'Open').click();
+    cy.wait(1000);
+
+    cy.create_orgBounty(bounty4);
+    cy.wait(1000);
+
+    cy.create_orgBounty(bounty5);
+    cy.wait(1000);
+
+    cy.create_orgBounty(bounty6);
+    cy.wait(1000);
+
+    cy.contains('Status').first().click();
+    cy.wait(1000);
+
+    cy.contains('label', 'Assigned').click();
+    cy.wait(1000);
+
+    cy.contains(bounty4.title);
+    cy.wait(1000);
+
+    cy.contains(bounty5.title);
+    cy.wait(1000);
+
+    cy.contains(bounty6.title);
+    cy.wait(1000);
+
+    cy.contains('label', 'Assigned').click();
+    cy.wait(1000);
+
+    cy.create_orgBounty(bounty7);
+    cy.wait(1000);
+
+    cy.create_orgBounty(bounty8);
+    cy.wait(1000);
+
+    cy.contains(bounty7.title).click();
+    cy.wait(1000);
+
+    cy.contains('Mark as Paid').click();
+    cy.wait(1000);
+
+    cy.contains('Next').click();
+    cy.wait(1000);
+
+    cy.contains('Skip and Mark Paid').click();
+    cy.wait(1000);
+
+    cy.contains('completed');
+
+    cy.get('body').click(0, 0);
+    cy.wait(1000);
+
+    cy.contains(bounty8.title).click();
+    cy.wait(1000);
+
+    cy.contains('Mark as Paid').click();
+    cy.wait(1000);
+
+    cy.contains('Next').click();
+    cy.wait(1000);
+
+    cy.contains('Skip and Mark Paid').click();
+    cy.wait(1000);
+
+    cy.contains('completed');
+
+    cy.get('body').click(0, 0);
+    cy.wait(1000);
+
+    cy.contains('Status').first().click();
+    cy.wait(1000);
+
+    cy.contains('label', 'Paid').click();
+    cy.wait(1000);
+
+    cy.contains(bounty7.title);
+    cy.wait(1000);
+
+    cy.contains(bounty8.title);
+    cy.wait(1000);
+
+    cy.contains('label', 'Paid').click();
+    cy.wait(1000);
+
+    cy.contains('Status').first().click();
+    cy.wait(1000);
```

Comment:
@Vayras log the activeUser out after the test.

Reply:
@elraphty done